src/app/forms/components/date/date.component.ts

```diff
@@ -45,6 +45,7 @@ export class DateComponent extends BaseControlComponent implements OnInit, OnDes
   public errors?: { error: boolean; date?: Date; errors?: { error: boolean; reason: string; index: number }[] };
   private dateFieldOrDefault?: Record<'hours' | 'minutes' | 'seconds', string | number>;
   public showError: boolean = false;
+  public minuteTouched: boolean = false;
```

Comment:
minute touched doesn't seem to be used anywhere?

Reply:
Yeah this needs to go, is residual from another change we attempted